ui/components/ui/new-network-info/new-network-info.js

```diff
@@ -61,7 +61,7 @@ export default function NewNetworkInfo() {
   }, [providerConfig.chainId]);
 
   useEffect(() => {
-    checkTokenDetection();
+    // checkTokenDetection();
```

Comment:
Was this change intentional?

Reply:
nope! reverted here: https://github.com/MetaMask/metamask-extension/pull/23130/commits/0e8ee1c6417c0d4bbb0be7222bd03e27d5a5d7ab